### ISSUE TITLE
Error usage with snowpack

### ISSUE THREAD

**Issue by rifaldhiaw** [nandorojo/swr-firestore#80]
I wanna use this lib in the snowpack App.   
But then, it throws
```
The requested module '../../web_modules/@nandorojo/swr-firestore.js' does not provide an export named 'useDocument'
```

Thanks

**Comment by nandorojo** [comment]
I haven't used Snowpack, so I'm not sure. The docs provided show how to use it in general React. There are also some GitHub issues for using with Next.js. However, I’m not familiar with Snowpack, so I can't help here. 

Please let me know if you figure it out!

**Comment by rifaldhiaw** [comment]
Most likely, the problem is because of the common Js module, and the snowpack needs it to be an ESM.
https://github.com/snowpackjs/snowpack/issues/440

however, when I tried the workaround that is written here, it still won't work 🤔

**Comment by mewhhaha** [comment]
I had the same issue. Never managed to resolve it, but tried with Vite which also uses esm modules but does prebundling of commonjs modules and had better results. However when I tried starting it I would get the error `cannot read length of undefined` as a result of `firebase.apps.length` in the `Fuego` class. It seems that the `import * as firebase from 'firebase/app'` statement is the culprit. I tried cloning the repo, and linking to a build where I had replaced the statement with `import firebase from 'firebase/app'` and it worked.
